Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -3,10 +3,18 @@ import Prelude
 
 public enum Colors {
   public static let black = Color.other("#121212")
-  public static let purple = Color.other("#974DFF")
   public static let blue = Color.other("#4CCCFF")
+  public static let gray300 = Color.other("#555555")
+  public static let gray400 = Color.other("#666666")
+  public static let gray650 = Color.other("#a8a8a8")
+  public static let gray850 = Color.other("#d8d8d8")
+  public static let gray900 = Color.other("#f6f6f6")
   public static let green = Color.other("#79F2B0")
```

Comment:
Should we get rid of `green`?

Reply:
we use it on the episode page for the TOC